open-api/src/testFixtures/java/org/apache/iceberg/rest/RESTServerExtension.java

```diff
@@ -32,17 +33,18 @@ public class RESTServerExtension implements BeforeAllCallback, AfterAllCallback
   private RESTCatalogServer localServer;
   private RESTCatalog client;
   private final Map<String, String> config;
+  private final boolean freePort;
```

Comment:
I would rather name this `findFreePort` since this is essentially what's happening